runtime/ops/os.rs

```diff
@@ -10,9 +10,11 @@ use deno_core::OpState;
 use serde::Serialize;
 use std::collections::HashMap;
 use std::env;
+use std::sync::atomic::AtomicI32;
 use std::sync::atomic::Ordering::Relaxed;
+use std::sync::Arc;
 
-pub fn init() -> Extension {
+pub fn init(maybe_exit_code: Option<Arc<AtomicI32>>) -> Extension {
```

Comment:
Does it have to be an `Arc`? It's only accessed from the same thread, I think.

Reply:
It has to, it's shares between all created threads (ie. web workers).